### ISSUE TITLE
LND GRPC QR Code is missing host part

### ISSUE THREAD

**Issue by ottosuess** [btcpayserver/btcpayserver#656]
I have access to two different BTCPay Server instances. The older one includes the host in the url for the lnd configuration in the QR Code, the newer one does not.

Without it I can't download the configuration within an app that does not now the full url of the server.

**BTCPayServer v1.0.3.16**
```
config=https://btcpayserver.x.com/lnd-config/123/lnd.config
```

**BTCPayServer v1.0.3.78**

```
config=/lnd-config/123/lnd.config
```

**Comment by pavlenex** [comment]
Both Zap and Pebble devs tested and confirm that this is a bug in BTCPay. Thanks.

**Comment by NicolasDorier** [comment]
yep, this is a bug, fixing it. Sorry!

**Comment by NicolasDorier** [comment]
Fixed in 1.0.3.80
